docs/es/docs/tutorial/query-params.md

```diff
@@ -132,7 +132,7 @@ No los tienes que declarar en un orden específico.
 Serán detectados por nombre:
 
 ```Python hl_lines="8  10"
-{!../../docs_src/query_params/tutorial004.py!}
+{*../../docs_src/query_params/tutorial004.py*}
```

Comment:
same comment as above, should be `{*../../docs_src/query_params/tutorial004.py hl[8,10] *}`